bind/symtab.go

```diff
@@ -130,6 +130,10 @@ func (s symbol) isMap() bool {
 	return (s.kind & skMap) != 0
 }
 
+func (s symbol) isSequence() bool {
```

Comment:
is this a function that tests for the Go-concept of a sequence or for the python concept of a sequence?

in other words, do we want to also test for maps ?

a quick grep in the code seems to indicate `isSequence` is used to determine whether we need to generate methods that implement the python's sequence protocol...